test/ScriptCs.Engine.Roslyn.Tests/CSharpScriptInMemoryEngineTests.cs

```diff
@@ -43,12 +43,12 @@ public void ShouldExposeExceptionThrownByScriptWhenErrorOccurs()
             [Fact]
             public void ShouldExposeExceptionThrownByCompilation()
             {
-                var scriptEngine = new RoslynScriptInMemoryEngine(new ScriptHostFactory(), new TestLogProvider());
+                var scriptEngine = new CSharpScriptInMemoryEngine(new ScriptHostFactory(), new TestLogProvider());
 
                 // Arrange
                 var lines = new List<string>
                 {
-                    "using Sysasdasdasdtem;"
+                    "Sysasdasdasdtem;"
```

Comment:
@adamralph that code (invalid namespace) actually does not throw any exceptions with this version of Roslyn - this is not an issue for us, cause we don't pass namespaces into code anyway (we extract them and populate `ScriptOptions` object itself)